packages/core/src/credentials/oidvc.ts

```diff
@@ -0,0 +1,71 @@
+import {OpenID4VCIClientV1_0_13} from '@sphereon/oid4vci-client';
+import {IWallet} from '../types';
+import {IDIDProvider} from '../did-provider';
+import {Alg} from '@sphereon/oid4vci-common';
+import {didServiceRPC} from '@docknetwork/wallet-sdk-wasm/src/services/dids';
+
+export async function acequireOpenIDCredentialFromURI({
+  didProvider,
+  uri,
+  getAuthCode,
+}: {
+  didProvider: IDIDProvider;
+  uri: string;
+  getAuthCode?: (authorizationURL: string) => Promise<string>;
+}) {
+  const [holderKeyDocument] = await didProvider.getDIDKeyPairs();
+
+  const queryString = uri.split('?')[1];
+  const params = new URLSearchParams(queryString);
+  const credentialOfferEncoded = params.get('credential_offer');
+  const credentialOfferDecoded = decodeURIComponent(credentialOfferEncoded);
+  const credentialOffer = JSON.parse(credentialOfferDecoded);
+  const scope = credentialOffer.credentials[0];
+  const format = 'ldp_vc';
+  const credentialTypes = scope.replace('ldp_vc:', '');
+
+  const client = await OpenID4VCIClientV1_0_13.fromURI({
+    uri: uri,
+    clientId: 'dock.wallet',
+    authorizationRequest: {
+      redirectUri: 'dock-wallet://credentials/callback',
+      clientId: 'dock.wallet',
```

Comment:
doesnt have to be dock.wallet, could also be `dock-wallet` - maybe this should be a config var (similar for redirect uri above?)

Reply:
I will handle that in the next PR, I want to unify that with the existing wallet-sdk configs injected by white label apps. Already logged a ticket for it https://dock-team.atlassian.net/browse/DCKM-580